services/viva-ms/lambdas/generateRecurringCaseHtml.js

```diff
@@ -100,10 +51,11 @@ export async function main(event, context) {
     return false;
   }
 
+  let answers = caseItem.forms[vivaCaseSSMParams.recurringFormId].answers;
```

Comment:
`vivaCaseSSMParams.recurringFormId` could be destructed because you use it in several places and I think this makes it a little more readable.

`const { recurringFormId } = vivaCaseSSMParams;`

Reply:
Done